### PR TITLE
fix: Font size has to be checked(refer design) [Our Investment Criteria]

### PR DESCRIPTION
**Pull request by Rajkumar-2k21**

### CODE REVIEW

**Comment by Rajkumar-2k21** [comment]
#5

File: css/bootstrap.min.css [hunk 1]
```diff
@@ -123,7 +123,7 @@ h6,.h6 {
 }
 
 p {
-    margin-top: 0;
+    margin-top: 17px;
```
Comment (balaji-sivasakthi) [inline css/bootstrap.min.css:126]: Why you add 17px for margin top

File: index.html [hunk 2]
```diff
@@ -609,59 +609,68 @@ <h2 class="text-center" style="padding-top:50px !important;color: white !importa
...
+                        <p style="font-size: small;">Healthcare, life sciences and pharma</p>
                     </div>
                 </div>
                 <div class="col-md-3">
                     <div class="cardlst2">
                         <img src="img/icon5.jpg" width="40px" />
-                        <h5>Social Impact</h5>
-                        <p>A clearly defined social impact thesis integrated with the business strategy</p>
+                        <h5> <b> Social Impact</b></h5>
+
```
Comment (balaji-sivasakthi) [inline index.html:621]: remove

File: index.html [hunk 3]
```diff
@@ -609,59 +609,68 @@ <h2 class="text-center" style="padding-top:50px !important;color: white !importa
...
+                        <p style="font-size: small;">A clearly defined social impact thesis integrated with the business strategy</p>
                     </div>
                 </div>
                 <div class="col-md-3">
                     <div class="cardlst2">
                         <img src="img/icon6.jpg" width="40px" />
-                        <h5>Path to Profitability</h5>
-                        <p>A sustainable business model with a well defined path to profitability</p>
+                        <h5> <b> Path to Profitability </b></h5>
+
```
Comment (balaji-sivasakthi) [inline index.html:629]: remove space

**Review by balaji-sivasakthi — COMMENTED** [review]
Things need to be change

- [x] remove unwanted spaces
- [x] remove the margin top in p
- [x] add margin as 0 default

**Review by balaji-sivasakthi — COMMENTED** [review]
Done

**Pull request merged**